### ISSUE TITLE
[1.19.2] Some Supplementaries recipes Missing - They'r visible in JEI but not with BOTH or EMI alone

### ISSUE THREAD

**Issue by Masangue** [emilyploszaj/emi#338]
Forge 1.19.2
EMI 1.0.23 
JEI 11.6.0.1018
Supplementaries 2.4.4

Seen it with recipes for colored presents, wich use a Supplementaries specific recipe type, they'r not visible.
It's the case for EMI alone but also when adding EMI to JEI, where they'r visible.

Specific recipes are for dying, trapping and removing color from presents

**Comment by emilyploszaj** [comment]
Can you screenshot some examples?

**Comment by Masangue** [comment]
Here some examples of the problem

Left is with EMI right is without it

![PresentColoring](https://github.com/emilyploszaj/emi/assets/55583833/3fccaec2-9243-4786-ab9e-64acff86f31f)
![PresentColorRemoving](https://github.com/emilyploszaj/emi/assets/55583833/16814438-eabc-4eed-87e0-9bc65e139bc4)
![SpikePotionInfusing](https://github.com/emilyploszaj/emi/assets/55583833/4bf80e09-eb20-4ba5-9fc4-d7b9f966ecfe)

Also for exemple how the present dyeing and pikes infusing recipes are made in the datapack part:

present_dye.json :
```
{
  "type": "supplementaries:present_dye",
  "conditions": [
    {
      "type": "supplementaries:flag",
      "flag": "present"
    }
  ]
}
```
bamboo_spikes_tipped.json :
```
{
  "type": "supplementaries:bamboo_spikes_tipped",
  "conditions": [
    {
      "type": "supplementaries:flag",
      "flag": "bamboo_spikes_tipped"
    }
  ]
}
```

**Comment by emilyploszaj** [comment]
Sorry for the delay here, I got down to the source of the issue and fixed it in 1.0.28, it affects several other mods but is now resolved.
